inference-engine/src/inference_engine/include/ie/ie_parallel.hpp

```diff
@@ -52,7 +52,7 @@ inline int parallel_get_num_threads() {
     return parallel_get_max_threads();
 }
 inline int parallel_get_thread_num() {
-    return tbb::this_task_arena::current_thread_index();
+    return (tbb::this_task_arena::max_concurrency() > 1) ? tbb::this_task_arena::current_thread_index() : 0;
```

Comment:
This line theoretically can be incorrect, because thread slot can be 1 not 0 but if you don't have any logic that related to thread slots everything should be fine as a workaround.

Reply:
We don't have thread slots related logic